components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/DBRetriever.java

```diff
@@ -48,7 +48,8 @@
     protected ApiMgtDAO apiMgtDAO = ApiMgtDAO.getInstance();
     protected EventHubConfigurationDto eventHubConfigurationDto = ServiceReferenceHolder.getInstance()
             .getAPIManagerConfigurationService().getAPIManagerConfiguration().getEventHubConfigurationDto();
-    private String baseURL = eventHubConfigurationDto.getServiceUrl();
+    private String baseURL = eventHubConfigurationDto.getServiceUrl() +
+            APIConstants.INTERNAL_WEB_APP_EP ;
```

Comment:
Shall we move this to single line?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9546/commits/348f1873a32b288469439cd23fb114098cc90e65